gunicorn/util.py

```diff
@@ -387,6 +387,8 @@ def import_app(module):
     try:
         app = eval(obj, mod.__dict__)
     except NameError:
+        exc_type, exc_value, exc_traceback = sys.exc_info()
+        traceback.print_exception(exc_type, exc_value, exc_traceback)
```

Comment:
Please either use star args to avoid locals or explicitly delete the traceback. See #1228.